styles/style.css

```diff
@@ -318,6 +340,111 @@
   animation: 1s ease-out rotate;
 }
 
+/* -------------Popup Window------------- */
+
+#overlay {

```

Comment:
- [ ] Good job so far 🥇, however, make sure you stuck to the design as much as possible (e.g., `font`, colors, text, `margins`) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=23-9&mode=design&t=W2s5ShWQ6gCg7kwW-0). as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) 👍.

![Screenshot 2023-07-25 175521](https://github.com/AlselwiAisha/Portfolio/assets/80458199/b0687058-84c2-446a-9dbd-d0aa62028335)


   - Kindly, fix the margin of the `Next project` button, and the `font-weight` of your project title 👍 
   - Also the size of your images.
   - I'll suggest you remove the `border-radius` of the popup 

![Screenshot 2023-07-25 180159](https://github.com/AlselwiAisha/Portfolio/assets/80458199/e9c47de6-9ae5-43ac-9e20-9bc555aee8b9)

 
![Screenshot 2023-07-25 180448](https://github.com/AlselwiAisha/Portfolio/assets/80458199/2ee893e5-861c-4053-ad15-d8764f72e064)